mlflow/pyfunc/__init__.py

```diff
@@ -1,16 +1,33 @@
 # -*- coding: utf-8 -*-
 
 """
-The ``mlflow.pyfunc`` module defines a generic :ref:`filesystem format <pyfunc-filesystem-format>`
-for Python models and provides utilities for saving to and loading from this format. The format is
-self contained in the sense that it includes all necessary information for anyone to load it and
-use it. Dependencies are either stored directly with the model or referenced via a Conda
-environment.
+The ``python_function`` model flavor serves as a default model interface for MLflow python models.
+Any MLflow python model is expected to be loadable as a ``python_function`` model.
+
+In addition, the ``mlflow.pyfunc`` module defines a generic :ref:`filesystem format
+<pyfunc-filesystem-format>` for Python models and provides utilities for saving to and loading from
+this format. The format is self contained in the sense that it includes all necessary information
+for anyone to load it and use it. Dependencies are either stored directly with the model or
+referenced via a Conda environment.
 
 The ``mlflow.pyfunc`` module also defines utilities for creating custom ``pyfunc`` models
 using frameworks and inference logic that may not be natively included in MLflow. See
 :ref:`pyfunc-create-custom`.
 
+.. _pyfunc-inference-api:
+
+*************
+Inference API
+*************
+
+Python function models are loaded as an instance of :py:class:`PyFuncModel
+<mlflow.pyfunc.PyFuncModel>` which is an MLflow wrapper around the model implementation and model
+metadata (MLmodel file). you can score the model by calling :py:func:`predict
```

Comment:
Can we link this `MLmodel` text to the Storage Format section? https://mlflow.org/docs/latest/models.html#storage-format